kernel/cPrimitives.mli

```diff
@@ -98,6 +105,7 @@ val kind : t -> args_red
 type 'a prim_type =
   | PT_int63 : unit prim_type
   | PT_float64 : unit prim_type
+  | PT_string : unit prim_type
```

Comment:
@rlepigre this is what was making serlib crash, unfortunately we need to use pierce here due to serialziaton limitations with gadts. See the fix in the PR.

Reply:
Cool, thanks for figuring this out!

Reply:
We have 8 piercing uses in the serlib codebase, I usually review them at Coq release time, by hand.

But it is indeed an error prone process, I'm trying to add tests for these 8 cases.